packages/ontario-frontend-cli/resources/templates/ontario-create-app/package.njk

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "rm -rf dist && eleventy",
     "debug": "rm -rf dist && DEBUG=* eleventy",
-    "serve": "eleventy --serve"{% if addESLint %},
+    "serve": "eleventy --serve",
+    "test": "npm list mocha chai || npm install mocha chai@4.4.1; mocha './src/tests/**/*.spec.js'"{% if addESLint %},
```

Comment:
Need to fix the `chai` version and fix the ES_Module error that stems from the latest `chai` package.

Reply:
https://github.com/chaijs/chai/issues/1561